File: app/src/debug/java/com/bopr/android/smailer/ui/DebugActivity.java
package com.bopr.android.smailer.ui;


import androidx.annotation.NonNull;

/**
 * For debug purposes.
 *
 * @author Boris Pronin (<a href="mailto:boprsoft.dev@gmail.com">boprsoft.dev@gmail.com</a>)
 */
public class DebugActivity extends BaseActivity {

    @NonNull
    @Override
    protected DebugFragment createFragment() {
        return new DebugFragment();
    }

}

File: app/src/debug/kotlin/com/bopr/android/smailer/ui/DebugActivity.kt
package com.bopr.android.smailer.ui

/**
 * For debug purposes.
 *
 * @author Boris Pronin ([boprsoft.dev@gmail.com](mailto:boprsoft.dev@gmail.com))
 */
class DebugActivity : BaseActivity() {

    override fun createFragment(): DebugFragment {
        return DebugFragment()
    }
}